src/spikeinterface/core/zarrextractors.py

```diff
@@ -0,0 +1,558 @@
+from __future__ import annotations
+
+from pathlib import Path
+from probeinterface import ProbeGroup
+
+import numpy as np
+
+from .baserecording import BaseRecording, BaseRecordingSegment
+from .basesorting import BaseSorting, BaseSortingSegment
+from .core_tools import define_function_from_class, check_json
+from .job_tools import split_job_kwargs
+
+try:
+    import zarr
+
+    HAVE_ZARR = True
+except ImportError:
+    HAVE_ZARR = False
```

Comment:
Given the fact that:
 * zarr will take a more important place in the spikeinterface with the sortingresult object
 * zarr is small and well maintained
 * zarr have frequent released
we could add zarr in hard dependencies one day no ?

Reply:
agree, adding to core

Reply:
done